cmd/edv-rest/startcmd/start.go

```diff
@@ -85,6 +95,8 @@ const (
 	tlsKeyFileEnvKey = "EDV_TLS_KEY_FILE"
 
 	dataVaultConfigurationStoreName = "data_vault_configurations"
+
+	sleep = 1 * time.Second
```

Comment:
You can simplify this to just `time.Second`

Reply:
fixed